app/src/main/res/layout/fragment_lists_for_account.xml

```diff
@@ -27,30 +26,18 @@
             android:id="@+id/listsView"
             android:layout_width="match_parent"
             android:layout_height="match_parent"
+            app:layoutManager="androidx.recyclerview.widget.LinearLayoutManager"
             android:visibility="gone"
             tools:visibility="visible" />
 
         <com.keylesspalace.tusky.view.BackgroundMessageView
             android:id="@+id/messageView"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
             android:src="@android:color/transparent"
             android:visibility="gone"
-            app:layout_constraintBottom_toBottomOf="parent"
-            app:layout_constraintEnd_toEndOf="parent"
-            app:layout_constraintStart_toStartOf="parent"
-            app:layout_constraintTop_toTopOf="parent"
             tools:src="@drawable/elephant_error"
             tools:visibility="visible" />
 
     </androidx.constraintlayout.widget.ConstraintLayout>
-
-    <Button
-        android:id="@+id/doneButton"
-        style="@style/TuskyButton.TextButton"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:layout_gravity="end"
-        android:text="@string/button_done" />
-
-</LinearLayout>
+</LinearLayout >
```

Comment:
This looks like it can be done with a single `FrameLayout`? Nested `LinearLayout`/`ConstraintLayout` is definitely too much.

Reply:
Done